### PR TITLE
Fix starting the vagrant environment on  Fedora 26

### PR DESCRIPTION
**Pull request by obnoxxx**
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/303)

### CODE REVIEW

File: vagrant/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+- name: restart network
+  systemd:
+    state: restarted
+    service: network
+  when: ansible_eth1.ipv4 is not defined
+- name: refresh inventory
```
Comment (jarrpa) [inline vagrant/roles/common/tasks/main.yml]: Add spaces between tasks.

**Comment by obnoxxx** [comment]
@JohnStrunk FYI
@jarrpa please review

File: vagrant/roles/common/tasks/main.yml (hunk 1 above)
Comment (obnoxxx) [inline vagrant/roles/common/tasks/main.yml]: done

**Comment by jarrpa** [comment]
LGTM

**Pull request merged**